test/server/policies.ts

```diff
@@ -0,0 +1,71 @@
+import { expect, test } from 'vitest'
+import { app } from './utils'
+
+test('policy list filtering', async () => {
```

Comment:
```suggestion
test('policy list with limit', async () => {
```

Reply:
also same as for other list tests, we should create some first to check that they are returned and that limit works